<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RocketStorageAuthWidget</class>
 <widget class="QWidget" name="RocketStorageAuthWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>164</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Scene Storage Authentication</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frameContent {
	background-color: white;
}
QFrame#frameButtons {
	border-top: 1px solid rgb(207, 207, 207);
}

QLabel, QPushButton, QLineEdit {
	font-family: &quot;Arial&quot;;
	font-size: 13px;
}

QLabel#labelError {
	color: red;
	background-color: white;
	padding: 8px;
}

QLabel#labelTitle {
	font-size: 15px;
	font-weight: bold;
	background-color: white;
	padding: 4px;
	padding-left: 10px;
	padding-bottom: 0px;
	color: rgb(59, 59, 59);
}

QPushButton#buttonAuthenticate, #buttonCancel {
	background-color: transparent;
	color: rgb(80, 80, 80);
	border: 1px solid grey;
	border: 0px;
	border-radius: 3px;
	font-family: &quot;Arial&quot;;
	font-weight: bold;
	min-width: 75px;
	min-height: 25px;
	max-height: 25px;
	padding-left: 5px;
	padding-right: 5px;
}

QPushButton#buttonAuthenticate::hover, #buttonCancel::hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(242, 244, 243, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(60, 60, 60);
	border: 1px solid grey;
}

QPushButton#buttonAuthenticate::pressed, #buttonCancel::pressed {
	background-color: rgba(200, 200, 200);
	border: 1px solid grey;
}

QPushButton#buttonAuthenticate:default, #buttonCancel:default {
	color: rgb(25,25,25);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>Administrator Scene Storage Authentication</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameContent">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="labelIcon">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="RocketPlugin.qrc">:/images/icon-update.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="accessKeyLabel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Storage Access Key</string>
          </property>
          <property name="text">
           <string>Access Key</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="accessKeyLineEdit">
          <property name="toolTip">
           <string>Storage Access Key</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="secretKeyLabel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Storage Secret Key</string>
          </property>
          <property name="text">
           <string>Secret Key</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="secretKeyLineEdit">
          <property name="toolTip">
           <string>Storage Secret Key</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelError">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameButtons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="labelAuthenticatingAnim">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAuthenticating">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAuthenticate">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Authenticate</string>
        </property>
        <property name="icon">
         <iconset resource="RocketPlugin.qrc">
          <normaloff>:/images/icon-check.png</normaloff>:/images/icon-check.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCancel">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="RocketPlugin.qrc">
          <normaloff>:/images/icon-cross.png</normaloff>:/images/icon-cross.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="RocketPlugin.qrc"/>
 </resources>
 <connections/>
</ui>
